examples/axisartist/demo_floating_axes.py

```diff
@@ -84,7 +84,9 @@ def setup_axes3(fig, rect):
 
     grid_locator2 = MaxNLocator(3)
 
+    # Set theta limits in degrees
```

Comment:
To be slightly bikesheddy, does this read as the local assignment _sets_ the limits, rather than these will be the limits that are passed in via the `extremes` kwargs to `GridHelperCurveLinear` below.

I may have been doing too much `pytest` work recently and am looking for magic everywhere...